spring-cloud-deployer-local/src/test/java/org/springframework/cloud/deployer/spi/local/LocalProcessDeployerIntegrationTests.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.deployer.spi.local;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.cloud.deployer.spi.process.ProcessDeployer;
+import org.springframework.cloud.deployer.spi.test.AbstractProcessDeployerTests;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Integration tests for LocalProcessDeployer.
+ *
+ * @author Eric Bottard
+ */
+@SpringApplicationConfiguration(classes = LocalProcessDeployerIntegrationTests.Config.class)
+public class LocalProcessDeployerIntegrationTests extends AbstractProcessDeployerTests {
```

Comment:
if the subclass is expected to have "Integration" in its name, we probably should go ahead and rename the base class too (also the base class' JavaDoc explicitly states "integration" tests)